doc/nrf/config_and_build/programming.rst

```diff
@@ -1,133 +1,88 @@
 .. _gs_programming:
 .. _programming:
+.. _programming_vsc:
+.. _programming_cmd:
 
-Building and programming an application
-#######################################
+Programming an application
+##########################
 
 .. contents::
    :local:
    :depth: 2
 
 |application_sample_definition|
 
-For additional information, check the user guide for the hardware platform that you are using.
-These user guides contain platform-specific instructions for building and programming.
-For example, see :ref:`ug_nrf5340_building` in the :ref:`ug_nrf5340` user guide for information about programming an nRF5340 DK, or :ref:`Programming precompiled firmware images <programming_thingy>` and :ref:`building_pgming` for information about programming a Thingy:91.
-
-.. note::
-   |application_sample_long_path_windows|
-
-.. _programming_vsc:
-
-Building with |VSC|
-*******************
-
-For instructions about building with the |nRFVSC|, see `How to build an application`_.
-If you want to build and program with custom options, read about the advanced `Custom launch and debug configurations`_.
-
-For information about how to flash and erase in the |nRFVSC|, read about the `Application-specific flash options`_.
-
-.. note::
-   |ncs_oot_sample_note|
-
-|output_files_note|
-
-.. _programming_cmd:
-
-Building on the command line
-****************************
-
-After completing the :ref:`command-line build setup <install_ncs>`, use the following steps to build |NCS| projects on the command line.
-
-1.    Open a terminal window.
-
-      If you have installed the |NCS| using the :ref:`legacy Toolchain Manager <gs_assistant>`, click the down arrow next to the version you installed and select :guilabel:`Open bash`.
-
-      .. figure:: ../installation/images/gs-assistant_tm_dropdown.png
-         :alt: The Toolchain Manager dropdown menu for the installed nRF Connect SDK version, cropped
-
-         The Toolchain Manager dropdown menu options
-
-#.    Go to the specific application directory.
+To program the :ref:`output build files <app_build_output_files>` to your device, use the steps for the development environment you have been using for the whole project.
```

Comment:
Also, the amount of build files in Output Build Files can be a bit overwhelming. Can I suggest that we state that zephyr.hex/merged.hex are the ones used by west flash by default here?